examples/generic-azure.yaml

```diff
@@ -673,3 +673,26 @@ groups:
           ## References
           - section 9.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
         scored: true
+      - id: 1.5.3
+        description: "<sid>adm user and its <sid> group exist and enabled"
+        audit: |
+          sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:upper:]' '[:lower:]')
+          if cat /etc/passwd | cut -d: -f1 | grep ${sid}adm; then echo userexists; fi
+          if ! cat /etc/passwd | grep -e "^${sid}adm" | cut -d: -f7 | grep "nologin"; then echo userenabled; fi
+          if cat /etc/group | cut -d: -f1,4 | grep "${sid}...:{$sid}adm"; then echo groupexists; fi
```

Comment:
@aleksei-burlakov  it looks like you are only checking if the sidadm user exists in any group, but the important part is to make sure sidadm is in the sapsys group. Please take a look at your test system, I would expect the sidadm user to be a member of sapsys, and this is what we want to check for.  Also good idea to verify with @angelabriel of course .

Reply:
> @aleksei-burlakov it looks like you are only checking if the sidadm user exists in any group, but the important part is to make sure sidadm is in the sapsys group. Please take a look at your test system, I would expect the sidadm user to be a member of sapsys, and this is what we want to check for.

For me that is a different check.
One check would be if the permissions (group membership) is as expected (see also haclient group). An other
check (this here) is to check only, if the users are defined locally. Maybe we need a layer to combine such check results to a combined result like "users are dined as expected".